docs/Installation Guide.md

```diff
@@ -6,8 +6,8 @@ This tutorial assumes that the user is installing and running the project under
 ## Table of Contents
 1. [Install git](https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#install-git)
 2. [Clone Project](https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#clone-project)
-2.5 [Install Python] (https://www.python.org/downloads/)
-2.6 [Install and make sure pip is working] (https://pip.pypa.io/en/latest/installing/)
+2.5 [Install Python] (https://github.com/systers/vms/blob/master/docs/Installation%20Guide.md#install-python)
```

Comment:
Mam, according to the issue #151 , it was expected to direct the link to Django girls. Directing it to main python website is also an option.